src/api/routes/interviews.ts

```diff
@@ -175,6 +182,34 @@ export async function createInterview(type: InterviewType, calibrationId: string
   });
 }
 
+const getInterviewerStats = procedure
+.use(authUser("MentorshipManager"))
+.output(z.array(z.object({
+  user: zUser,
+  interviews: z.number(),
+})))
+.query(async () =>
+{
+  const mentors = await db.User.findAll({
+    attributes: userAttributes,
+    include: userInclude,
+    where: { roles: { [Op.contains]: ["Mentor"] }, }
+  });
+
+  const stats = await Promise.all(mentors.map(async mentor => {
```

Comment:
It will be faster to read count all the interviewfeedback rows grouped by interviewids at once, and then compose the return result in memory.

Reply:
the current code will issue many DB queries which can be very slow.